python/py-more-itertools/Portfile

```diff
@@ -37,10 +37,30 @@ if {${name} ne ${subport}} {
                             sha256  38a936c0a6d98a38bcc2d03fdaaedaba9f412879461dd2ceff8d37564d6522e4 \
                             size    67359
         depends_lib-append  port:py${python.version}-six
-    }
 
-    if {${python.version} < 37} {
         python.pep517       no
+    } elseif {${python.version} < 36} {
```

Comment:
this only applies to Python 3.5, so I would personally just write it as ``} elseif {${python.version} == 35} {`

Reply:
Here it is